pkg/controller/healthz.go

```diff
@@ -16,15 +18,21 @@ import (
 
 var (
 	rl = rate.NewLimiter(rate.Every(time.Minute), 1)
+
+	mHealthzBuildInfo = stats.Int64("build_info", "Current build information", "1")
```

Comment:
observability.MetricRoot + "/build_info" for first arg, stats.UnitDimensionless for last arg.